lib/include/openamp/rpmsg_virtio.h

```diff
@@ -58,6 +58,35 @@ void rpmsg_return_buffer(struct rpmsg_virtio_device *rpmsgv, void *buffer,
 int rpmsg_virtio_enqueue_buffer(struct rpmsg_virtio_device *rpmsgv,
 				void *buffer, unsigned long len,
 				unsigned short idx);
+
+static inline unsigned int rpmsg_virtio_get_role(struct rpmsg_virtio_device *rvdev)
+{
+	return rvdev->vdev->role;
```

Comment:
since role is RPMSG_MASTER or RPMSG_SLAVE, it seems like that should belong in rpmsg_virtio_device and not virtio_device.